pages/common/git-blame-someone-else.md

```diff
@@ -0,0 +1,8 @@
+# git blame-someone-else
+
+> Blame someone else for your bad code.
+> More information: <https://github.com/jayphelps/git-blame-someone-else>.
+
+- Change the commiter and author of a commit:
+
+`git blame-someone-else "{{author <email>}}" {{commit}}`
```

Comment:
```suggestion
`git blame-someone-else "{{author someone@example.com}}" {{commit}}`
```

Reply:
From the GIF in the README, this is actually correct, as it species the email for the `git log`

```
commit 8d854c2d78cb98afdb9f5a73240e06393260b327 (HEAD -> master, origin/master, origin/HEAD)
Author: Jay Phelps <hello@jayphelps.com>
Date:   Tue Jun 12 21:06:40 2018 -0400

```

Reply:
Ok, so
```suggestion
`git blame-someone-else "{{author <someone@example.com>}}" {{commit}}`
```